src/Error/JsonApiExceptionRenderer.php

```diff
@@ -0,0 +1,205 @@
+<?php
+namespace Crud\Error;
+
+use Cake\Core\Configure;
+use Cake\Datasource\ConnectionManager;
+use Cake\Error\Debugger;
+use \Neomerx\JsonApi\Document\Error;
```

Comment:
Fixed in https://github.com/FriendsOfCake/crud/pull/445/commits/6a70b299a063ead55c934455f5fb491c811426bb